.github/workflows/ci-test.yml

```diff
@@ -366,3 +366,38 @@ jobs:
           if [[ "$success" != "true" ]]; then
             exit 1;
           fi
+
+  perf-benchmarks:
```

Comment:
nit - Can you move this block above ci-test-success (line 328)?  This would make the validation signal (for Bors) easier to find and read.